addon/components/editor-plugins/template-variable-card.js

```diff
@@ -19,6 +19,7 @@ export default class EditorPluginsTemplateVariableCardComponent extends Componen
     const config = getOwner(this).resolveRegistration('config:environment');
```

Comment:
another question here, why are we not using the plugin config mechanism anymore?
reminder you can add an object with the shape 
```
{ name: string, options: any }
```
in the plugin array argument to the RdfaEditor component, and the plugin will get the options object as its second argument to the initialize function
Genuine question, not trying to say one is better than the other

Reply:
I'm following the design decision of putting on the environment anythign that doesn't need to be changed from instance of the editor to another instance and on the plugin config the things that need to be changed. Not sure if it's the correct decision and we should move everything to the editor way